force-app/main/default/objects/Deal_Support_Process__c/recordTypes/Enterprise_Training.recordType-meta.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<RecordType xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Enterprise_Training</fullName>
+    <active>true</active>
```

Comment:
and let's not update the developername in these scenarios. I'm guessing it would break a decent amount of references throughout the repo.

Reply:
The API name is now restored to its previous state, "Training".